healthcareai/common/helpers.py

```diff
@@ -6,4 +11,33 @@ def count_unique_elements_in_column(dataframe, column_name):
     :return: number of target classes
     """
     uniques = dataframe[column_name].unique()
-    return len(uniques)
\ No newline at end of file
+    return len(uniques)
+
+
+def calculate_random_forest_mtry_hyperparameter(number_of_columns, model_type):
+    """
+    Calculates a reasonable list of mtry hyperparameters (Number of variables available for splitting at each tree node.)
```

Comment:
This is trivial, but let's stay under 80 chars (and follow PEP8 generally). 

As we formalize more with CAFE, perhaps we can sit down sometime and set up PEP8, pyflakes, pylinter checks in PyCharm for everyone...